polynote-frontend/polynote/ui/component/about.ts

```diff
@@ -185,12 +185,14 @@ export class About extends FullScreenModal {
                         ]);
                         const actionsEl = div([], [
                             iconButton(['start'], 'Start kernel', 'power-off', 'Start').click(() => {
-                                this.publish(new KernelCommand(path, 'kill'));
+                                this.publish(new KernelCommand(path, 'start'));
                                 getKernelStatuses();
+                                setTimeout(() => getKernelStatuses(), 5000); // TODO: this should be event-driven!
```

Comment:
this `setTimeout` hack is annoying. We should revisit this when we get to the UI rethink we talked about a while ago